frontend/src/App.css

```diff
@@ -1,20 +1,25 @@
 #app {
-    display: flex;
-    height: 100%;
+    display: grid;
```

Comment:
We dropped the idea of using `CCS grid` a while ago, because it's not supported by our browser support matrix.

Reply:
![](https://media.giphy.com/media/xT1XGWbE0XiBDX2T8Q/giphy.gif)

Reply:
Can we maybe align our support matrix on the grid feature? I don't remember how we chose those browsers to support, so maybe that's a dumb question. :-)

Reply:
OK, I'll have a look and PR the branch with an update to the CSS file. Or the browser matrix. :-)